### PR TITLE
About section

### PR DESCRIPTION
**Pull request by principles31**
Portfolio: mobile version: About section 
About Section was created, 
this project contains
- index.html
- about, work and index CSS
- Additional images
- Linter were enabled to check for bugs

### CODE REVIEW

**Comment by principles31** [comment]
@chaw-bot I do not understand these comments kindly elaborate because I implement margin's where I need to implement.
 Kindly implement the margins (top, bottom, left, right) defined in the design.

File: index.html [hunk 1]
```diff
@@ -41,12 +42,165 @@ <h1>Emmanuel Mensah</h1>
...
+                        Take a look of my works. If you like what you see 
+                        and have a project you need coded, don't hesitate 
+                        and contact me.
+                    </p>
+                </div>
+
+                <a class="text_button text_body_font about_button">
+                    <p> Get my CV</p>
+                    <div class="download_image"></div>  
+                </a>
```
[the remark this answers is not in the document]
Comment (principles31) [reply index.html:161]: @chaw-bot would be happy to know the CSS syntax for Pressed

File: index.html [hunk 2]
```diff
@@ -41,12 +42,165 @@ <h1>Emmanuel Mensah</h1>
...
+                    </div>
+                    <div class="about_row about_row_3">
+                        <ul class="skill_set"> 
+                            <li class="about_skills text_body_font"> Skills </li> 
+                            <li class="li_class"> Github</li>
+                            <li class="li_class"> Terminal</li>
+                        </ul>
+                        <div class="about_img about_img_3"></div>
+                    </div>
+                </div>
```
[the remark this answers is not in the document]
Comment (principles31) [reply index.html:191]: @chaw-bot I have implemented the display:flex

**Review by sja-thedude — APPROVED** [review]
Hi @principles31, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**